beets/autotag/mb.py

```diff
@@ -475,6 +478,59 @@ def album_info(release):
         info.genre = '; '.join(g[0] for g in sorted(genres.items(),
                                                     key=lambda g: -g[1]))
 
+    # Additional online sources if configured and MusicBrainz provides them.
+    # FIXME This check is always True because of config_default.yaml.
+    if ('url_rels' in config['musicbrainz'].keys() and
+            release.get('url-relation-list')):
+        discogs_url, deezer_url = None, None
+        spotify_url, bandcamp_url, beatport_url = None, None, None
+        fetch_discogs, fetch_spotify, fetch_bandcamp = False, False, False
+        fetch_beatport, fetch_deezer = False
```

Comment:
@JOJ0 needs ", False"

Reply:
Seems to be my "Thanks" a couple of days ago didn't get through. So again: Thanks a lot! Also fixed in original commit! :-)